### ISSUE TITLE
Encoding HTML entities

### ISSUE THREAD

**Issue by thehands79** [ralphjsmit/laravel-seo#73]
Hi, there's an encoding inconsistency with HTML entities:

**Config:**
`'homepage_title' => 'Buy & Sell',`

**Not encoded:**
`<title>Buy & Sell</title>`

**Encoded:**
```
<meta property="og:title" content="Buy &amp; Sell">
<meta name="twitter:title" content="Buy &amp; Sell">
```

**Comment by ralphjsmit** [comment]
Hi @thehands79, thanks for your message! I think this is because the output of homepage_title is escaped in Blade using `{{ }}`. Is this an error you think?

**Comment by thehands79** [comment]
Hi, thanks for the reply.

I would classify it as a bug as the same configuration value is being displayed differently within the same context.

Either encode HTML entities in all HTML tags or not. Encoding some but not others doesn't make sense.

Is it something clients can control?

**Comment by ralphjsmit** [comment]
Hey! Fixed in #75.
